webpack/assets/javascripts/react_app/components/Layout/index.js

```diff
@@ -0,0 +1,25 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as actions from './LayoutActions';
+import reducer from './LayoutReducer';
+
+import Layout from './Layout';
+
+// map state to props
+const mapStateToProps = ({ layout }) => ({
+  items: layout.items,
+  isLoading: layout.isLoading,
+  activeMenu: layout.activeMenu,
```

Comment:
Instead of calculating the activeMenu in the action/reducer, you can filter it out by a selector here.